crates/re_renderer/src/renderer/depth_cloud.rs

```diff
@@ -0,0 +1,378 @@
+                entries: vec![
+                    wgpu::BindGroupLayoutEntry {
+                        binding: 0,
+                        visibility: wgpu::ShaderStages::VERTEX | wgpu::ShaderStages::FRAGMENT,
+                        ty: wgpu::BindingType::Buffer {
+                            ty: wgpu::BufferBindingType::Uniform,
+                            has_dynamic_offset: false,
+                            min_binding_size: (std::mem::size_of::<gpu_data::DepthCloudInfoUBO>()
+                                as u64)
+                                .try_into()
+                                .ok(),
+                        },
+                        count: None,
+                    },
+                    wgpu::BindGroupLayoutEntry {
+                        binding: 1,
+                        visibility: wgpu::ShaderStages::VERTEX,
+                        ty: wgpu::BindingType::Texture {
+                            sample_type: wgpu::TextureSampleType::Float { filterable: false },
+                            view_dimension: wgpu::TextureViewDimension::D2,
+                            multisampled: false,
+                        },
+                        count: None,
+                    },
+                ],
+            },
+        );
+
+        let pipeline_layout = pools.pipeline_layouts.get_or_create(
+            device,
+            &PipelineLayoutDesc {
+                label: "depth_cloud_rp_layout".into(),
+                entries: vec![shared_data.global_bindings.layout, bind_group_layout],
+            },
+            &pools.bind_group_layouts,
+        );
+
+        let shader_module = pools.shader_modules.get_or_create(
+            device,
+            resolver,
+            &ShaderModuleDesc {
+                label: "depth_cloud".into(),
+                source: include_file!("../../shader/depth_cloud.wgsl"),
+            },
+        );
+
+        let render_pipeline = pools.render_pipelines.get_or_create(
+            device,
+            &RenderPipelineDesc {
+                label: "depth_cloud_rp".into(),
+                pipeline_layout,
+                vertex_entrypoint: "vs_main".into(),
+                vertex_handle: shader_module,
+                fragment_entrypoint: "fs_main".into(),
+                fragment_handle: shader_module,
+                vertex_buffers: smallvec![],
+                render_targets: smallvec![Some(ViewBuilder::MAIN_TARGET_COLOR_FORMAT.into())],
+                primitive: wgpu::PrimitiveState {
+                    topology: wgpu::PrimitiveTopology::TriangleList,
+                    ..Default::default()
+                },
+                depth_stencil: ViewBuilder::MAIN_TARGET_DEFAULT_DEPTH_STATE,
+                multisample: wgpu::MultisampleState {
+                    // We discard pixels to do the round cutout, therefore we need to
+                    // calculate our own sampling mask.
+                    alpha_to_coverage_enabled: true,
```

Comment:
Alpha to coverage still enabled here but killed-off in the shader. Commit to one or the other! :)